### PR TITLE
Suggestion for episode

### PR DESCRIPTION
**Pull request by Hellowlol**
Possible uses
````
show = show.search('The Blacklist')[0]

show.episode(season=1, episode=1)
<Episode:116263:The.Freelancer>

show.season(1)
<Season:116260/children:Season.1>
````
Im not sure about the api.

### CODE REVIEW

File: plexapi/video.py [hunk 1]
```diff
@@ -189,12 +189,16 @@ def seasons(self):
...
 
-    def season(self, title):
+    def season(self, title=None):
         """Returns a Season
 
         Args:
-            title (str): fx Season1
+            title (str, int): fx Season 1
         """
+        title = str(title)
```
Comment (mjs7231) [inline plexapi/video.py]: I think using isinstance(title, int) might be a better choice here than converting it to a string then checking isdigit()

File: plexapi/video.py [hunk 2]
```diff
@@ -207,9 +211,18 @@ def episodes(self, watched=None):
         leavesKey = '/library/metadata/%s/allLeaves' % self.ratingKey
         return utils.listItems(self.server, leavesKey, watched=watched)
 
-    def episode(self, title):
-        path = '/library/metadata/%s/allLeaves' % self.ratingKey
-        return utils.findItem(self.server, path, title)
+    def episode(self, title=None, season=None, episode=None):
```
Comment (mjs7231) [inline plexapi/video.py]: I like the idea of title or season & episode. We should raise an exception at the end of this function if we fall through the if statements explaining that they need to specify both season & episode for the second case to work.

File: plexapi/video.py [hunk 3]
```diff
@@ -268,25 +281,38 @@ def episodes(self, watched=None):
...
 
-        
+
         """
         childrenKey = '/library/metadata/%s/children' % self.ratingKey
         return utils.listItems(self.server, childrenKey, watched=watched)
 
-    def episode(self, title):
-        """Find a episode with a matching title.
+    def episode(self, title=None, season=None, episode=None):
```
Comment (mjs7231) [inline plexapi/video.py]: Keeping season as an argument here seems incorrect, probably should be removed.

File: plexapi/video.py (hunk 2 above)
Comment (Hellowlol) [inline plexapi/video.py]: Agree. Should we raise not found or just return None if we don't find any episode too?

File: plexapi/video.py (hunk 3 above)
Comment (Hellowlol) [inline plexapi/video.py]: i wasn't sure if I should add season, I only did so episode in show and season accept the same kwargs. I'll remove it

File: plexapi/video.py (hunk 1 above)
Comment (Hellowlol) [inline plexapi/video.py]: Yep

**Pull request merged**